src/services/storage/index.ts

```diff
@@ -37,19 +39,25 @@ export class Storage implements StorageDriver {
 			return this.storage._driver
 		}
 		public async get(id: string): Promise<StoredUrl> {
+			logger.debug(`Running get with ${id}`)
 			return this.driver.url.get(id)
 		}
 		public async delete(id: string): Promise<void> {
+			logger.debug(`Running delete with ${id}`)
 			return this.driver.url.delete(id)
 		}
 		public async deleteOverdue(timespanMs: number): Promise<number> {
+			logger.debug(`Running deleteOverdue with ${timespanMs}`)
 			return this.driver.url.deleteOverdue(timespanMs)
 		}
 		public async edit(id: string, url: string): Promise<StoredUrl> {
+			logger.debug(`Running edit with ${id} and ${url}`)
 			return this.driver.url.edit(id, url)
 		}
 
 		public async save(url: string): Promise<StoredUrl> {
+			// logger.debug(`Start save with ${url}`)
+			// סליחה על העברית, התוצאה שמוחזרת היא לכאורה רגישה, אז כרגע זה בקומיט
```

Comment:
This is actually not sensitive. You can remove the Hebrew and add the logger.

**What's sensitive?**
Database credentials, admin token, personal information (which we currently don't have).

**What's _not_ sensitive?**
ids, URLs (without tokens in the query etc.), IPs and anything else.